packages/react-network/src/manager.ts

```diff
@@ -1,12 +1,14 @@
 import {StatusCode, CspDirective, Header} from '@shopify/network';
 import {EffectKind} from '@shopify/react-effect';
+import {Cookie, UniversalCookies} from '@shopify/react-cookie';
```

Comment:
isn't this a circular dependency? Should the server cookie manager live in `@shopify/react-network`?

Reply:
Yeah also agree with this